src/library/coverartutils.cpp

```diff
@@ -214,16 +210,16 @@ CoverArt CoverArtUtils::selectCoverArtForTrack(
     }
 
     if (bestInfo != NULL) {
-        art.image = QImage(bestInfo->filePath());
-        if (!art.image.isNull()) {
-            art.info.source = CoverInfo::GUESSED;
-            art.info.type = CoverInfo::FILE;
+        QImage image(bestInfo->filePath());
+        if (!image.isNull()) {
+            coverInfoRelative.source = CoverInfo::GUESSED;
+            coverInfoRelative.type = CoverInfo::FILE;
             // TODO() here we may introduce a duplicate hash code
-            art.info.hash = CoverArtUtils::calculateHash(art.image);
-            art.info.coverLocation = bestInfo->fileName();
-            return art;
+            coverInfoRelative.hash = CoverArtUtils::calculateHash(image);
```

Comment:
Redundant "CoverArtUtils::". There are more occurrences of this redundant prefix in the file.